### ISSUE TITLE
Some tests failed after installing Spacy

### ISSUE THREAD

**Issue by lingvisa** [explosion/spaCy#3502]
## How to reproduce the problem

I followed the install instruction "Compile from source" exactly on my Mac OS and everything works fine without an issue. However, when I run the test by executing:
```bash
python -m pytest spacy/
...

1779 passed, 302 skipped, 56 xfailed, 10 warnings in 23.05 seconds, 11 warnings in 54.82 seconds
```
As you can see, 302 tests are skipped and 56 failed. Is this normal? or some of the failures indicate something wrong in the installation?

## Your Environment

* Operating System: MacOS
* Python Version Used: python 3.6.5
* spaCy Version Used: source
* Environment Information: installed the requirements

**Comment by ines** [comment]
Sorry if this was confusing – but I think the 56 tests are actually **x**failed tests, so **expected** failures. Basically, those are tests that describe things that should work but currently don't. In pytest, those are usually marked with the `@pytest.mark.xfail` decorator. 

I think in our test suite, most of the xfailing tests are in the tokenizer tests and describe very specific (and sometimes tricky) examples that the tokenizer currently doesn't get right. There are also a few other rough edges in the library that we've added xfailing tests for. This lets us keep a record of those test cases without having our actual test suite fail.

The 302 skipped tests are tests that aren't run by default – usually those are tests that are interesting but not crucial and often also slow (like [this one](https://github.com/explosion/spaCy/blob/master/spacy/tests/tokenizer/test_naughty_strings.py)). pytest also skips tests for languages that require additional dependencies to be installed (like Japanese or Russian).

**TL;DR:** Looks like your installation is fine and the tests run as expected 🙂 If pytest doesn't report any explicit tests failures (marked with an `F` in the overview and usually also listed below the test results summary), everything is usually fine and running as expected.

**Comment by lingvisa** [comment]
Thanks ines. I am closing it thus.

**Comment by lingvisa** [comment]
close, since it is not bug.
